packages/graphql-yoga/src/plugins/useGraphiQL.ts

```diff
@@ -14,24 +14,18 @@ export type GraphiQLOptions = {
    * will use its own default query.
    */
   defaultQuery?: string
-  /**
-   * Whether to open the variable editor by default. Defaults to `true`.
-   */
-  defaultVariableEditorOpen?: boolean
   /**
```

Comment:
we should mark these as `@deprecated` cause if we just remove it can likely break someone.

Reply:
Thank you for pointing that. I've updated